### ISSUE TITLE
How does this sample profile have a static code

### ISSUE THREAD

**Issue by Alavi1412** [zerobias/telegram-mtproto#74]
every time I enter my number in this nodeJS app, Telegram sends a different code each time but the sample use auth.signIn methods with a single code.
1.how can I get a code just like that?
2.how can I received the codes when the developer mode is true?(it sent the code the first time to my phone and I used signUp method but It didn't send the code anymore)

**Comment by goodmind** [comment]
@Alavi1412 

1. You can't. This is test server only
2. You should use `signIn` method after you created account with `signUp`

**Comment by Alavi1412** [comment]
@goodmind 
I have used `signIn` but it doesn't send code to my phone and the `signIn` method  would fail

**Comment by goodmind** [comment]
@Alavi1412 what error?

**Comment by Alavi1412** [comment]
@goodmind 
the problem is the connection is finished. when I use the sing in method I should use a code but I don't know what is the code.
surely the error is: INVALID_PHONE_CODE

**Comment by goodmind** [comment]
@Alavi1412 you should enter code from SMS

**Comment by Alavi1412** [comment]
@goodmind 
I don't receive SMS.
and I can't used sendSMS method. it says undefined method

**Comment by goodmind** [comment]
@Alavi1412 use `auth.sendCode`

**Comment by Alavi1412** [comment]
@goodmind 
it is my problem. I didn't receive SMS code.

**Comment by goodmind** [comment]
@Alavi1412 what you mean

**Comment by goodmind** [comment]
@Alavi1412 do you still can't receive SMS?

**Comment by Alavi1412** [comment]
@goodmind 
I solved the problem.
When we logged in an app in telegram, for another device, telegram doesn't send sms to your phone and send the code to the previews app. and when I tested it in test mode, I haven't an app in test mode to receive code. so I found two solutions:
1.using 'resendCode' will send SMS to my phone,
2.open the desktop telegram and go to setting and type `testmode` , then the telegram will open in testmode and I can get the code from there.
